lib/widgets/theme.dart

```diff
@@ -219,6 +244,11 @@ class DesignVariables extends ThemeExtension<DesignVariables> {
 
   // Not named variables in Figma; taken from older Figma drafts, or elsewhere.
   final Color atMentionMarker;
+  final Color bgContextMenu;
+  final Color contextMenuCancelBg;
+  final Color contextMenuCancelText;
+  final Color contextMenuItemBg;
+  final Color contextMenuItemText;
```

Comment:
Thanks. I forgot that these became named variables from Figma. Also, still kept `contextMenuCancelBg` in the file because of a [previous discussion](https://chat.zulip.org/#narrow/stream/48-mobile/topic/bottom.20sheet.20redesign.20-.20.23F853/near/1909523), but left the necessary comments.